netbox/utilities/templatetags/builtins/filters.py

```diff
@@ -152,22 +154,30 @@ def content_type_id(model):
 # Rendering
 #
 
+
 @register.filter('markdown', is_safe=True)
-def render_markdown(value):
+def render_markdown(value, classes=""):
     """
     Render a string as Markdown. This filter is invoked as "markdown":
+    Args: optional coma separated list of classes to add to the generated div
```

Comment:
```suggestion
    Args: 
        optional: Comma separated list of classes to add to the generated div
```

Typo in comma and change to match format used other places for Arg lists.